api/src/test/java/com/epam/pipeline/acl/configuration/ServerlessConfigurationApiServiceTest.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2017-2020 EPAM Systems, Inc. (https://www.epam.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.epam.pipeline.acl.configuration;
+
+import com.epam.pipeline.controller.vo.configuration.RunConfigurationVO;
+import com.epam.pipeline.entity.configuration.RunConfiguration;
+import com.epam.pipeline.manager.configuration.RunConfigurationManager;
+import com.epam.pipeline.manager.configuration.ServerlessConfigurationManager;
+import com.epam.pipeline.manager.pipeline.runner.ConfigurationProviderManager;
+import com.epam.pipeline.manager.security.AuthManager;
+import com.epam.pipeline.security.acl.AclPermission;
+import com.epam.pipeline.test.acl.AbstractAclTest;
+import com.epam.pipeline.test.creator.configuration.ConfigurationCreatorUtils;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.security.access.AccessDeniedException;
+import org.springframework.security.test.context.support.WithMockUser;
+
+import javax.servlet.http.HttpServletRequest;
+
+import static com.epam.pipeline.test.creator.CommonCreatorConstants.ID;
+import static com.epam.pipeline.test.creator.CommonCreatorConstants.TEST_STRING;
+import static com.epam.pipeline.util.CustomAssertions.assertThrows;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Matchers.anyLong;
+import static org.mockito.Mockito.doReturn;
+
+@SuppressWarnings("PMD.TooManyStaticImports")
```

Comment:
Could you please remove the rule from `config/pmd/pmd-ruleset.xml` and then remove `@SuppressWarnings` since is not relevant anymore. We decided to get rid of this pmd rule because it doesn't seem to help the code style but just making everything harder.